src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,49 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	public static final int MAX_CAPACITY = 12;
+	public static final int EMPTY = -1;
+	public static final int ERROR = -1;
+	public static final int MAX_INDEX = 11;
```

Comment:
This constant is related to MAX_CAPACITY. Changing only one leads to an error (inconsistency).